modules/services/github-runner/service.nix

```diff
@@ -90,15 +90,15 @@ in
           let
             configure = pkgs.writeShellApplication {
               name = "configure-github-runner-${name}";
-              text = ''
+              text = /*bash*/''
                 export RUNNER_ROOT
 
                 args=(
                   --unattended
                   --disableupdate
                   --work ${escapeShellArg workDir}
                   --url ${escapeShellArg cfg.url}
-                  --labels ${escapeShellArg (concatStringsSep "," cfg.extraLabels)}
+                  --labels "${escapeShellArg (concatStringsSep "," cfg.extraLabels)}"
```

Comment:
> @Enzime Could you instead wrap the argument to `--labels` with double quotes?

Looks like `escapeShellArg` is adding its own quote. See #1085

Reply:
My apologies for having introduced this. @Enzime looks like the original suggestion with a `#shellcheck disable` directive was the correct way to go.